assets/dev/js/editor/regions/panel/pages/elements/favorites/commands/create.js

```diff
@@ -0,0 +1,24 @@
+import CommandsBase from './base';
+
+export class Create extends CommandsBase {
+	/**
+	 * @inheritDoc
+	 */
+	apply( args ) {
+		_.each( args.widgets, ( widget ) => {
+			const widgetCache = this.getWidgetCache( widget );
+
+			if ( undefined !== widgetCache ) {
+				widgetCache.categories.push( this.getCategoryName() );
+			}
+		} );
```

Comment:
> Try to avoid using `_` in the future we will remove all the uses of `_`.
> 
> ```js
> args.widget.forEach( ( widget ) => { //.... } )
> ```

Alright, happy to know that good old JS can be used here.